web/src/engine/websites/decorators/Liliana.ts

```diff
@@ -0,0 +1,42 @@
+//For websites similar to MangaSect, Manga347, ManhuaPlus, ManhuaGold,  MangaReader.to,
+
+export const mangaPath ='/all-manga/{page}/';
+export const queryMangaTitleFromURI = 'div.anisc-detail .manga-name';
+export const queryMangas = 'div.item div.manga-detail .manga-name a';
+export const queryChapters = [
+    'ul#chapters-list li a.item-link',
+    'ul#myUL li.chapter a',
+].join(',');
+
+export const queryPagesScript = `
+    new Promise(resolve => {
+
+        function parseResults(data) {
+            const dom = new DOMParser().parseFromString(data, 'text/html');
+            let nodes = [...dom.querySelectorAll('div.separator')];
+
+            //sort if needed
+            if (nodes[0].hasAttribute('data-index')) {
+                nodes = nodes.sort(function (a, b) {
```

Comment:
No need to re-assign, sort mutates the original array and just returns a reference.